package.json

```diff
@@ -58,6 +58,7 @@
     "gulp-util": "^3.0.7",
     "gzip-js": "~0.3.2",
     "jest": "^15.1.1",
+    "jest-cli": "^15.1.1",
```

Comment:
It’s best to make it as close to `master` as we can in terms of linting setup. It would make sense to me to have 32f5b034ed229d048f76ae74e18d270edc801dbf and similar changes cherry-picked.